packages/cmf/src/onError.js

```diff
@@ -71,7 +71,7 @@ function hasReportFeature() {
  * @param {Error} error instance of Error
  */
 function report(error, options = {}) {
-	if (ref.SENTRY_DSN) {
+	if (ref.SENTRY_DSN || getCurrentHub().getClient() !== undefined) {
```

Comment:
there are many places where we test ref.SENTRY_DSN to consider the sentry setup.
Not sure what is the best but here the fix is imcomplete. 
For example hasReportFeature will report false but sentry is available... so the Error Component will display wrong info.